feature/bgp/policybase/otg_tests/link_bandwidth_test/README.md

```diff
@@ -0,0 +1,149 @@
+# RT-7.5: BGP Policy - Set Link Bandwidth Community
+
+## Summary
+
+Configure bgp policy to add statically configured BGP link bandwidth
+communities to routes based on a prefix match.
+
+## Testbed type
+
+* https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+* Testbed configuration - Setup BGP sessions and prefixes.
+  * Generate config for 2 DUT and ATE ports where:
+    * DUT port 1 to ATE port 1.
+    * DUT port 2 to ATE port 2.
+  * Configure ATE port 1 with a BGP session to DUT port 1.
+    * Advertise ipv4 and ipv6 prefixes to DUT port 1 using the following communities:
+      * prefix-set-1 with 2 ipv4 and 2 ipv6 routes without communities.
+      * prefix-set-2 with 2 ipv4 and 2 ipv6 routes with communities `[ "100:100" ]`.
+      * prefix-set-3 with 2 ipv4 and 2 ipv6 routes with extended communities `[ "link-bandwidth:100:0" ]`.
+
+* RT-7.5.1 - Validate bgp sessions and traffic
+  * For IPv4 and IPv6 prefixes:
+    * Observe received prefixes at ATE port-2.
+  * Generate traffic from ATE port-2 to ATE port-1.
+  * Validate that traffic can be received on ATE port-1 for all installed
+    routes.
+
+* RT-7.5.2 - Validate adding link-bandwidth ext-community-sets using OC model release 3.x
+  * Configure the following extended community sets on the DUT:
+    (prefix: `routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set`)
+    * Create an ext-community-set named 'linkbw_0' with:
+      * ext-community-member = [ "link-bandwidth:100:0" ]
+    * Create an ext-community-set named 'linkbw_1M' with members as follows:
+      * ext-community-member = [ "link-bandwidth:100:1M" ]
+    * Create an ext-community-set named 'linkbw_2G' with members as follows:
+      * ext-community-member = [ "link-bandwidth:100:2G" ]
+    * Create an community-set named 'regex_match_as100' with members as follows:
+      * ext-community-member = [ "^100:.*$" ]
```

Comment:
This do not match ASN 100. because ^100 == 0x0064, it mathes in best case: 
-Transitive Two-Octet AS-Specific Extended Community ( 0x00)
- Unassigned sub-type (0x64)
- any ASN
- any local value

Reply:
The bgp community type string format isn't a bitmap directly to the bits on the wire.  The second : delimited field   is the asn.  The device is expected to decode bgp packets and  match the string 100 with the community decoded from the wire, including (optionally) the ASN.  In the [OC bgp-type for link-bandwidth](https://github.com/openconfig/public/blob/8387d23434be80b419375c4cc80c45546163e2e2/release/models/bgp/openconfig-bgp-types.yang#L605-L623), specifying ASN is required (based on the regex pattern defined).  

Only a few NOS directly expose the ability to statically configure the bandwidth and the ASN directly.  Here is an [example from EOS](https://www.arista.com/en/support/toi/eos-4-24-1f/14534-bgp-ucmp)